app/app.go

```diff
@@ -410,9 +424,41 @@ func NewRollapp(
 	)
 	transferIBCModule := ibctransfer.NewIBCModule(app.TransferKeeper)
 
+	wasmDir := filepath.Join(homePath, "wasm")
+	wasmConfig, err := wasm.ReadWasmConfig(appOpts)
+	if err != nil {
+		panic(fmt.Sprintf("error while reading wasm config: %s", err))
+	}
+
+	// The last arguments can contain custom message handlers, and custom query handlers,
+	// if we want to allow any custom callbacks
+	availableCapabilities := strings.Join(AllCapabilities(), ",")
+	app.WasmKeeper = wasmkeeper.NewKeeper(
+		appCodec,
+		keys[wasmtypes.StoreKey],
+		app.GetSubspace(wasm.ModuleName),
+		app.AccountKeeper,
+		app.BankKeeper,
+		app.StakingKeeper,
```

Comment:
FYI, the staking module on the rollapp is not related to consensus
not sure what the `app.StakingKeeper` is used for, but the block proposer will be in the sequencer module.
just in case it required somewhere

Reply:
the app.Staking keeper is mainly used for query stuff, not for consensus so I think it would be fine when put it in wasmKeeper